go/arm/Dockerfile.tmpl

```diff
@@ -4,10 +4,10 @@ ARG TAG_EXTENSION=''
 FROM --platform=linux/amd64 ${REPOSITORY}/golang-crossbuild:${VERSION}-base${TAG_EXTENSION} as stage-amd64
 
 RUN dpkg --add-architecture arm64 \
-    && apt update -y --no-install-recommends \
-    && apt upgrade -y --no-install-recommends \
-    && apt full-upgrade -y --no-install-recommends \
-    && apt install -qq -y --no-install-recommends \
+        && apt update -y --no-install-recommends \
```

Comment:
**nit**: as reviewer, all these ident changes make the review a bit complicated 🙏